### PR TITLE
Handle inline style changes without removing existing ones

### LINKED ISSUE #17: Bold+Italic doesn't work together
If we type something like this
  ```*** bold and italic ***```
It doesn't work.

### PR DESCRIPTION
**Pull request by juliankrispel**
The main focus of this pr is a rewrite of the inline styling logic.

1. Before this - we would replace all text we want to style.
2. With this change we instead remove the markdown delimiting characters and then apply the chosen style.

There are also a bunch of updates to the test cases. This works as expected now and fixes #17 as well as #25

### CODE REVIEW

File: src/modifiers/__test__/handleInlineStyle-test.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 /* eslint-disable no-unused-vars */
 
-import { expect } from "chai";
+//import { expect } from "chai";
 import Draft, { EditorState, SelectionState } from "draft-js";
 import handleInlineStyle from "../handleInlineStyle";
 
-describe("handleInlineStyle", () => {
+describe.only("handleInlineStyle", () => {
```
Comment (mxstbr) [inline src/modifiers/__test__/handleInlineStyle-test.js]: `.only`

**Comment by juliankrispel** [comment]
Whoops. Thx 😂

**Comment by mxstbr** [comment]
Works great!

**Comment by mxstbr** [comment]
Oops jow there's conflicts, my bad. Mind cleaning them up? (am on mobile)

**Comment by juliankrispel** [comment]
I don’t mind 👌

**Comment by juliankrispel** [comment]
@mxstbr thanks for resolving <3

**Pull request merged**